llvm/lib/Target/AMDGPU/AMDGPUIGroupLP.cpp

```diff
@@ -1726,6 +1749,8 @@ bool MFMAExpInterleaveOpt::applyIGLPStrategy(
   // In loop MFMAs
   auto MFMAInLoop = MFMAPipeCount - (MFMAEnablement * 2);
   auto MFMALoopCount = MFMAInLoop / MFMARatio;
+  auto VALUOps =
```

Comment:
Maybe just double-check whether any of these divs need a check for div-by-zero.

Reply:
Added a couple checks to be safe, but we should be good